esp/esp/users/models/__init__.py

```diff
@@ -613,6 +613,9 @@ def getFirstClassTime(self, program):
                 return sections[0].meeting_times.order_by('start')[0]
     getFirstClassTime.depend_on_row('program.StudentRegistration', lambda reg: {'self': reg.user})
 
+    def canPassPhaseZero(self, program):
```

Comment:
can we call this `can_skip_phase_zero` instead?  "pass" could mean either override or just has already filled it out.  (Although honestly I'd just inline it -- no need to define a wrapper for this.  But that's a stylistic preference, either way is ok.)